v7/import_blogger/import_blogger.py

```diff
@@ -191,12 +194,19 @@ def import_item(self, item, out_folder=None):
             # If no content is found, no files are written.
             content = self.transform_content(content)
 
-            self.write_metadata(os.path.join(self.output_folder, out_folder,
-                                             slug + '.meta'),
-                                title, slug, post_date, description, tags)
-            self.write_content(
-                os.path.join(self.output_folder, out_folder, slug + '.html'),
-                content)
+            regex = re.compile('/(?P<year>\d{4})/(?P<month>\d{2})/(?P<slug>.*)')
+            match = regex.search(link_path)
+            if match:
+                year, month, slug = match.group('year'), match.group('month'), match.group('slug')
```

Comment:
Using split seems much simpler. Blogger allows custom slugs but url structure is fixed.